src/main/java/in/mohamedhalith/dto/LoginDTO.java

```diff
@@ -1,14 +1,20 @@
 package in.mohamedhalith.dto;
 
+import javax.validation.constraints.Pattern;
+
 import lombok.AllArgsConstructor;
 import lombok.Data;
 
 @Data
 @AllArgsConstructor
 public class LoginDTO {
-	
+
+	@Pattern(regexp = "[A-Za-z0-9]{7,30}", message = "Invalid username")
 	private String username;
+	
+	@Pattern(regexp = "[A-Za-z0-9]{8,30}", message = "Invalid password")
```

Comment:
Don't put pattern for login credentials, because the hacker easily find out the password and username of others.so please remove pattern here.

Reply:
Will remove it in the next update